vc/pokecrystal11_vc.patch.template

```diff
@@ -0,0 +1,698 @@
+;ROM:BB2A1                 and     $B            
+;ROM:BB2A3                 jr      z, unk_B29C   
+;ROM:BB2A5                 and     8             
+;ROM:BB2A7                 jr      nz, unk_B2AA  
+;ROM:BB2A9                 ret    
+; 000bb2a5h: E6 08 20 01       
+; 000b92a3h: E6 08 20 01                                     ; ? .                           
+; change "and 8" to "and 0"                                                      
+[print forbid 4]                                                  
+Mode = 1                                                          
+Address = {hex @ 4}                                          
+Fixcode={patch}            
+
+
+;ROM:401D6                 call    unk_50A5                                         
+;ROM:401D9                 ld      hl, $FFA9  
+;ROM:401DC                 ld      a, [hl]    
+;ROM:401DD                 and     2          
+;ROM:401DF                 jr      nz, unk_1F8
+;ROM:401E1                 ld      a, [hl]    
+;ROM:401E2                 and     1          
+;ROM:401E4                 jr      nz, unk_1EE
+;ROM:401E6                 call    unk_4562   
+;ROM:401E9                 ret     nc         
+;ROM:401EA                 call    unk_4114   
+;ROM:401ED                 ret       
+; -----6666666666ddddddddd88888----0xc6d8 no ..............Mem Write: pc32 = 0x4109b addr = 0xc6d8 value = 0x0
+
+;00040266h: 7E E6 01 20 08                                  ; ~? .                                 
+[print forbid 5]                                                                                           
+Mode = 6                                                                                          
+Type = 0                                                                                          
+Address = {hex @ 4}                                                                            
+MemAddress={hex hJoyPressed}                                                                                 
+Fixcode={db NO_INPUT}                                                                                     
+ConditionType = 0                                                                                 
+ConditionValueA = {dws_ wWindowStackPointer wWindowStackPointer+1 wMenuJoypad wMenuSelection wDexArrowCursorPosIndex hJoyPressed hJoyPressed hJoyPressed hJoyPressed}                      
+ConditionValueB = {dws_ ==                  ==                    ==          ==             ==                      !=          !=          !=          !=         }                      
+ConditionValueC = {dws_ 0xa1                0xdb                  A_BUTTON    0x00           0x03                    D_DOWN      D_UP        B_BUTTON    NO_INPUT   }                      
```

Comment:
This is the one for the pokedex. the `wDexArrowCursorPosIndex` value corresponds to an entry in `DexEntryScreen_ArrowCursorData`.